src/Boids.hs

```diff
@@ -51,9 +54,14 @@ guideBoidToPoint :: Point -> Boid -> Boid
 guideBoidToPoint guidePoint boid = guideBoidToVector direction boid
     where direction = fromPoints (boidPosition boid) guidePoint
 
+
+isWithinView :: Boid -> Boid -> Bool
+isWithinView boid1 boid2 = angleVV (unitVectorAtAngle $ boidHeading boid1) (fromPoints (boidPosition boid1) (boidPosition boid2)) <= viewAngle
```

Comment:
Naming `agnleVV`'s result "azimuth" or something will make this easier to understand, both by breaking complexity and keeping the lines at sane length.